spec/models/vote_spec.rb

```diff
@@ -1,32 +1,31 @@
-require 'test_helper'
-
-class VoteTest < ActiveSupport::TestCase
-  test "is valid" do
+require 'rails_helper' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.